style/js/shield_defs.js

```diff
@@ -140,8 +140,17 @@ export function loadShields(shieldImages) {
 
   shields["US:US:Truck"] = banneredShield(shields["US:US"], ["TRK"]);
   shields["US:US:Spur"] = banneredShield(shields["US:US"], ["SPUR"]);
+  shields["US:US:Connector"] = banneredShield(shields["US:US"], ["CONN"]);
```

Comment:
After a Slack discussion, we discovered that the "Connector" banner has been removed from MUCTD for a long time, and the sole remaining route relation with this tagging was incorrectly tagged.  It has since been converted into an unsigned US:US:Business route.
```suggestion
  shields["US:US:Spur"] = banneredShield(shields["US:US"], ["SPUR"]);
```

Reply:
I looked through all connector routes from [this Wikipedia list](https://en.wikipedia.org/wiki/List_of_special_routes_of_the_United_States_Numbered_Highway_System) and while most of them are unsigned, connector routes for [US 19](https://en.wikipedia.org/wiki/Special_routes_of_U.S._Route_19#Bryson_City_connector), [US 29 (Williamston)](https://en.wikipedia.org/wiki/Special_routes_of_U.S._Route_29#Williamston_connector), and [US 117](https://en.wikipedia.org/wiki/U.S._Route_117#Calypso_Connector) are signed.